pkg/converter/source/virtualmachine.go

```diff
@@ -54,13 +59,13 @@ func (v *VirtualMachineSource) GetLabelsFromClient(_ client.Client) map[string]s
 	return map[string]string{config.ExternalEntityLabelCloudVPCKey: v.Status.VirtualPrivateCloud}
 }
 
-// GetExternalNode returns external node/controller associated with VirtualMachine.
-func (v *VirtualMachineSource) GetExternalNode(_ client.Client) string {
+// GenerateExternalNodeName returns nephe-controller as the ExternalNode Name.
+func (v *VirtualMachineSource) GenerateExternalNodeName(_ client.Client) string {
```

Comment:
This should be VM Name, not Nephe, otherwise agented policy will be received by nephe-controller